lib/packwerk/application_load_paths.rb

```diff
@@ -28,7 +28,9 @@ def extract_application_autoload_paths
           .select { |railtie| railtie.is_a?(Rails::Engine) }
           .push(Rails.application)
           .flat_map do |engine|
-          (engine.config.autoload_paths + engine.config.eager_load_paths + engine.config.autoload_once_paths).uniq
+          (engine.config.autoload_paths + engine.config.eager_load_paths + engine.config.autoload_once_paths)
```

Comment:
I find this a bit hard to read. Can you move the contamination into a separate descriptive variable?

Reply:
I've updated to be more readable.